features/stores.feature

```diff
@@ -93,6 +93,47 @@ Feature: Stores
       }
       """
 
+  Scenario: List stores
+    Given the fixtures files are loaded:
+      | sylius_channels.yml |
+      | stores.yml          |
+    And the user "bob" is loaded:
+      | email      | bob@coopcycle.org |
+      | password   | 123456            |
+    And the user "bob" has role "ROLE_ADMIN"
+    And the user "bob" is authenticated
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I add "Accept" header equal to "application/ld+json"
+    And the user "bob" sends a "GET" request to "/api/stores"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the JSON should match:
+      """
+      {
+        "@context": "/api/contexts/Store",
+        "@id": "/api/stores",
+        "@type": "hydra:Collection",
+        "hydra:member": [
+          {
+            "@id": "/api/stores/@string@",
+            "id": "@integer@.greaterThan(0)",
+            "@type": "http://schema.org/Store",
+            "name": "@string@",
+            "enabled": true,
+            "address": {"@*@":"@*@"},
+            "timeSlot": "/api/time_slots/1",
+            "timeSlots": @array@,
+            "prefillPickupAddress": @boolean@,
+            "weightRequired": @boolean@,
+            "packagesRequired": @boolean@,
+            "multiDropEnabled": @boolean@
+          },
+          @...@
+        ],
+        "hydra:totalItems": 9
+      }
```

Comment:
I'd suggest checking at least store names to make sure that `Deleted store` is not in the list, not merely `hydra:totalItems`

Something like:

```
 "hydra:member": [
          {
            "@id": "/api/stores/@string@",
            "id": "@integer@.greaterThan(0)",
            "@type": "http://schema.org/Store",
            "name": "Acme",
            "@*@":"@*@"
          },
          {
            "@id": "/api/stores/@string@",
            "id": "@integer@.greaterThan(0)",
            "@type": "http://schema.org/Store",
            "name": "Acme2",
            "@*@":"@*@"
          },
        ]
```

Reply:
That's a good one, will do in a next PR! :raised_hands:

Reply:
Done as part of PR: https://github.com/coopcycle/coopcycle-app/pull/1958
At commit: https://github.com/coopcycle/coopcycle-web/pull/4929/commits/8f814a0a32cdce320d104b6c75acde0e3b923748